### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by 20221174**
Add .gitignore for resolving the issue.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.o
```
Comment (melonhong) [inline .gitignore:1]:
```suggestion
*.o
*.out
*.exe
```

**Review by melonhong — CHANGES_REQUESTED** [review]
I've leave some comments. Please check them and address it before merge.

File: .gitignore (hunk 1 above)
Comment (20221174) [reply .gitignore:1]: Ok.

**Comment by 20221174** [comment]
Modify .gitignore.

**Review by melonhong — APPROVED** [review]
Well done!

**Pull request merged**